server/storeProfile.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once 'config.php';
+
+storeProfile(_$POST);
+
+function storeProfile($data) {
+  GLOBAL $DB_CONN;
+  $response = array();
+  $response['success'] = false;
+
+  $DB_CONN -> beginTransaction();
+
+  try {
+    $clientInsert = "INSERT INTO vita.user
+    (
+      firstName,
+      lastName,
+      phone,
+      email
+    )
+    VALUES
+    (
+      ?,
+      ?,
+      ?,
+      ?
+    );";
+    $userparams = array(
+      $data['firstName'],
+      $data['lastName'],
+      $data['email'],
+      $data['phone']
+    );
+    $stmt = $DB_CONN -> prepare($userInsert)
+    $stmt -> execute($userParams);
+
+    $userabilityID = $DB_CONN -> lastInsterId();
+
+    $abilityInsert = "INSERT INTO vita.userability
+    (
+    languageSkills,
+    taxSkills,
+    taxSkillsType
+  )
+  VALUES
+  (
+    ?,
+    ?,
+    ?
+  );";
+
+  $userabilityParams = array(
+    $data['languageSkills'],
+    $data['taxSkills'],
+    $data['taxSkillsType']
+  );
+
+  $stmt = $DB_CONN -> prepare($abilityInsert)
+  $stmt -> execute($userabilityParams);
+
+  $appointmentID = $DB_CONN -> lastInsterId();
+
+  $appointmentInsert = "INSERT INTO vita.servicedappointment
+  (
+    shiftsAvailable,
+    shiftsWorking,
+    siteId
+  )
+  VALUES
+  ?,
+  ?,
+  ?
+);";
+$appointmentParams = array(
+  $data['shiftsAvailable'],
+  $data['shiftsWorking'],
+  $data['taxSkillsType']
+);
+$stmt = DB_CONN -> prepare($appointmentInsert)
+$stmt -> execute($appointmentParams);
+
+  }
+getProfile(_$GET);
```

Comment:
This isn't how this stuff works. If you have multiple functions in a file that will get called, you need to do the `switch` statement type thing at the top just like Schmidt does. PHP code is imperative, it will run top to bottom and call any method that is in the main scope. So your current setup will call both `getProfile` and `storeProfile` whenever someone calls this file.